### DISCUSSION TITLE
Can I remove all demo users in internal_users.yml

### DISCUSSION THREAD

**Discussion by 4ss3mbl3rV** [wazuh/wazuh-docker#768] [Q&A]
Hi there!
I'm not pretty sure about the demo user that comes with the container project. Can I delete it all and only leave the admin one?

**Comment by c-bordon** [comment]
Deleting these users can cause problems for the stack, unless you replace these users with new users with the same permissions and configure the stack components with these users.

What you can do is change the password for one that you consider more secure, I share the password management documentation where you will find the password-tool to make the changes you think are necessary:
https://documentation.wazuh.com/current/user-manual/user-administration/password-management.html

In the case of deployment with docker, to change the passwords you can follow this documentation: https://documentation.wazuh.com/current/deployment-options/docker/wazuh-container.html#change-the-password-of-an-existing-user
